fe1-web/src/features/social/navigation/SocialMediaNavigation.tsx

```diff
@@ -47,7 +47,7 @@ const iconSelector =
  * This class manages the social media navigation and creates the corresponding navigation bar.
  */
 const SocialMediaNavigation = () => {
-  const [currentUserPopTokenPublicKey, setCurrentUserPopTokenPublicKey] = useState(
+  const [currentUserPublicKey, setCurrentUserPublicKey] = useState(
```

Comment:
could we rename this to clarify that we're talking about PoP tokens and not the (constant) public key ?
maybe `currentPopToken` would be enough ?

I seem to recall the long name, at the time, was because @Xelowak felt that `PopToken` would imply having the private key there as well. That could still be a valid point. Just giving you the history 😄

Reply:
Either of those works for me, whichever you think is better 😅